spark-stream-ingest/src/main/python/requirements.txt

```
@@ -0,0 +1,4 @@
+pandas==1.2.4
+cortex-python==6.1.0
+delta-spark==1.1.0
+pyyaml==5.3.1
```

```suggestion
pyyaml==6.0.0
```
We need to use the latest released version of libraries it minimizes the security risks and makes it easier to maintain the examples over time.  The PyYaml might be incompatible, but when we start with older libraries it makes it harder to migrate safely